src/views/CreateCourseSite.vue

```diff
@@ -327,6 +326,7 @@ export default {
       this.alertScreenReader('Loading courses and sections')
 
       const semester = (this.adminMode === 'bySectionId' ? this.currentAdminTerm : this.currentSemester)
+      this.selectedTerm = semester
```

Comment:
@lyttam - does the line above fix a functional bug?

Reply:
@johncrossman yes, `this.selectedTerm` was being initialized too early (before the value was available), causing the term name to be initially blank on the blue tab button:
![image](https://github.com/ets-berkeley-edu/ripley/assets/12432859/ca41357f-ff9a-466c-8f97-eeeaca8f4dab)